Add Azure provision support (#56)
kyma-incubator/hydroform: examples/azure/README.md

# Provision an Azure cluster

## Overview

This example shows you how to use Hydroform to provision an Azure cluster.

## Installation

### Configure Azure

> To provision a Azure cluster you need the client ID and client secret of your azure subscription stored in a TOML file as follows:

Reviewer: ```suggestion
To provision an Azure cluster you need the **CLIENT_ID** and **CLIENT_SECRET** of your Azure subscription stored in a TOML file:
```

```toml
CLIENT_ID = "<my-client-id>"
CLIENT_SECRET = "<my-client-secret>"
```

### Run the example

1. To provision a new cluster on Azure, go to the `hydroform` directory and run:

```bash
go run ./examples/azure/main.go -p {resource_group_name} -c /{path/to/credentials.toml}
```

> 2. On the Azure portal, go to **Home** > **All Resources** to see your cluster on the list.

Reviewer: ```suggestion
2. In the Azure portal, go to **Home** > **All Resources** to see your cluster on the list.
```

> 4. Export **KUBECONFIG** environment variable pointing to the `kubeconfig` file generated by running the example. This will allow access to the cluster.

Reviewer: ```suggestion
3. Export the **KUBECONFIG** environment variable pointing to the `kubeconfig` file generated by running the example. This will allow you to access the cluster.
```
